Source/Core/DolphinQt2/HotkeyScheduler.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <thread>
+
+class MainWindow;
+
+class HotkeyScheduler
+{
+public:
+  explicit HotkeyScheduler(MainWindow* parent);
+
+  void Start();
+  void Stop();
+
+private:
+  void Run();
+
+  bool m_stop_requested;
```

Comment:
There are no simulatinous writes happening here. So I think it should be fine this way?